google-codelab/google_codelab.js

```diff
@@ -681,9 +698,19 @@ class Codelab extends HTMLElement {
     this.renderDrawer_();
 
     if (document.location.hash) {
-      this.setAttribute(SELECTED_ATTR, document.location.hash.substring(1));
+      const h = document.location.hash.substring(1);
+      if (h !== '0') {
```

Comment:
I hope we'll move away from the hash part at some point. One idea could be to use an additional URL param like `?step=1`, or maybe `/codelabs/my-codelab-id/1`.
Not today though, just a random comment.

Reply:
yeah that would be totally doable. I'm just doing feature parity for now, but I also would like to get rid of the # and use ? instead. / would be a tad harder because we would need to url rewrite on the server.